test/integration/clustertrustbundles/field_selector_test.go

```diff
@@ -31,10 +31,6 @@ import (
 )
 
 func TestCTBSignerNameFieldSelector(t *testing.T) {
-	// KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE allows for APIs pending removal to not block tests
-	// TODO: Remove this line once certificates v1alpha1 types to be removed in 1.32 are fully removed
```

Comment:
cc @aaron-prindle @liggitt: We're extending the lifecycle of this API rather than removing it. Is this fine?